docs/reference/commandline/run.md

```diff
@@ -42,6 +42,7 @@ parent = "smn_cli"
       --help=false                  Print usage
       -i, --interactive=false       Keep STDIN open even if not attached
       --ipc=""                      IPC namespace to use
+      --isolation="default"         Container isolation level
```

Comment:
Be consistent

Container isolation technology

or

Container kernel isolation technology